### PR TITLE
Update docs with curated packages installation during cluster creation

### PR DESCRIPTION
**Pull request by a-cool-train**
*Issue #, if available:*
- Include curated packages installation in the docs.

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/local-environment/_index.md [hunk 1]
```diff
@@ -70,46 +70,116 @@ To install the EKS Anywhere binaries and see system requirements please follow t
...
-   🎉 Cluster created!
-   ------------------------------------------------------------------------------------------------------------------------------
-   The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription.
-   ------------------------------------------------------------------------------------------------------------------------------
-   Installing helm chart on cluster	{"chart": "eks-anywhere-packages", "version": "0.2.0-eks-a-v0.0.0-dev-build.3842"}
-   secret/aws-secret created
-   job.batch/eksa-auth-refresher created
-   package.packages.eks.amazonaws.com/my-harbor created
-   ```
+2. Create Cluster: Create your cluster either with or without curated packages:
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/local-environment/_index.md:81]:
```suggestion
1. Create Cluster: Create your cluster either with or without curated packages:
```

File: docs/content/en/docs/getting-started/local-environment/_index.md [hunk 2]
```diff
@@ -70,46 +70,116 @@ To install the EKS Anywhere binaries and see system requirements please follow t
...
+      job.batch/eksa-auth-refresher created
+      ```
+   - Cluster creation with optional curated packages
+
+     {{% alert title="Note" color="primary" %}}
+   * It is *optional* to install curated packages as part of the cluster creation.
+   * `eksctl anywhere version` version should be later than `v0.9.0`.
+   * Post-creation installation and detailed package configurations can be found [here.]({{< relref "../../tasks/packages" >}})
+   * The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/)
+     {{% /alert %}}
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/local-environment/_index.md:121]: Can we move all this to a step before the cluster creation. There should be a step of configuring this stuff the way the prod version are documented

**Review by TerryHowe — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/getting-started/local-environment/_index.md [hunk 3]
```diff
@@ -70,46 +70,116 @@ To install the EKS Anywhere binaries and see system requirements please follow t
...
+         metallb                 0.12.1-b9e4e5d941ccd20c72b4fec366ffaddb79bbc578  
+         emissary                3.0.0-a507e09c2a92c83d65737835f6bac03b9b341467
+         ```
+
+      * Generate a curated-packages config
+
+        The example shows how to install the `harbor` package from the [curated package list]({{< relref "../../reference/packagespec" >}}).
+         ```bash
+         eksctl anywhere generate package harbor --source registry --kube-version 1.23 > packages.yaml
+         ```
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/local-environment/_index.md:149]: This should probably be a step

File: docs/content/en/docs/getting-started/local-environment/_index.md [hunk 4]
```diff
@@ -70,46 +70,116 @@ To install the EKS Anywhere binaries and see system requirements please follow t
...
+        Example command output
+         ```                 
+         Package                 Version(s)                                       
+         -------                 ----------                                       
+         harbor                  2.5.0-4324383d8c5383bded5f7378efb98b4d50af827b
+         hello-eks-anywhere      0.1.1-a217465b3b2d165634f9c24a863fa67349c7268a   
+         harbor                  2.5.1-a217465b3b2d165634f9c24a863fa67349c7268a   
+         metallb                 0.12.1-b9e4e5d941ccd20c72b4fec366ffaddb79bbc578  
+         emissary                3.0.0-a507e09c2a92c83d65737835f6bac03b9b341467
+         ```
```
Comment (TerryHowe) [inline docs/content/en/docs/getting-started/local-environment/_index.md:142]: This looks like a step

File: docs/content/en/docs/getting-started/local-environment/_index.md (hunk 2 above)
Comment (a-cool-train) [reply docs/content/en/docs/getting-started/local-environment/_index.md]: There already was documentation looks like. I added a small note.

File: docs/content/en/docs/getting-started/local-environment/_index.md (hunk 4 above)
Comment (a-cool-train) [reply docs/content/en/docs/getting-started/local-environment/_index.md]: Removed this.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**